Docs 6867 visualizer visualization doc move (#59)
mulesoft/docs-visualizer: modules/ROOT/pages/customize-your-canvas.adoc

Visualizer editors can save defined views so that any Visualizer user can access them.
When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.

> You can customize layers, tags, and display names in the Visualizer UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).

Reviewer: ... in the Architecture Visualization UI

Author: ```suggestion
You can customize layers, tags, and display names in the Architecture Visualization UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).
```

Visualizer editors can save defined views so that any Visualizer user can access them.
When you close Anypoint Visualizer and reload the canvas later, the views you last accessed are displayed.

> You can customize layers, tags, and display names in the Architecture Visualization UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).

Reviewer 2: We don't refer to these as UI in the actual descriptions of each visualization.

> You can customize layers, tags, and display names in the Architecture Visualization UI or by setting a runtime property. If you are using a Mule runtime patch that was released after December 12, 2019, you can use all unicode characters with UTF-8 encoding (such as international letters, symbols, and numbers).

Reviewer 2: GLOBALLY replace "Mule runtime" with either "Mule" or "Mule runtime engine."

== Layers

> You can use one of the default layers provided by Anypoint Visualizer, or add a custom layer to create a visual representation of the apps in your network. You can use layers across Anypoint Visualizer, but note that you can only define layers in the xref:architecture-visualization.adoc[Architecture Visualization].

Reviewer 2: only?
```suggestion
You can use one of the default layers provided by Anypoint Visualizer, or add a custom layer to create a visual representation of the apps in your network. You can use layers across Anypoint Visualizer, but note that you can define layers only in the xref:architecture-visualization.adoc[Architecture Visualization].
```

To automate the classification of nodes into layers, configure the property `anypoint.platform.visualizer.layer=<name>`.

For more information, see xref:layers.adoc[Use Layers in Anypoint Visualizer].

== Tags

> You can add tags to API, application, proxy, or other services, so you can use the tags as filters to adjust your view. You can use tags across Anypoint Visualizer, but note that you can only define tags in the xref:architecture-visualization.adoc[Architecture Visualization].

Reviewer 2: ```suggestion
You can add tags to API, application, proxy, or other services and then use these tags as filters to adjust your view. You can use tags across Anypoint Visualizer, but note that you can define tags only in the xref:architecture-visualization.adoc[Architecture Visualization].
```

> To automate adding tag(s) to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>`.

Reviewer 2: ```suggestion
To automate adding tags to a node, use the property `anypoint.platform.visualizer.tags=<name>, <name>`.
```

For more information, see xref:use-tags-in-visualizer.adoc[Use Tags in Visualizer].

[[display-name]]
== Display Name

> You can edit the display name of a service to provide a more readable, explanatory title. You can use display names across Anypoint Visualizer, but note that you can only define display names in the xref:architecture-visualization.adoc[Architecture Visualization].

Reviewer 2: GLOBAL
```suggestion
You can edit the display name of a service to provide a more readable, explanatory title. You can use display names across Anypoint Visualizer, but note that you can define display names only in the xref:architecture-visualization.adoc[Architecture Visualization].
```
